api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -199,7 +201,7 @@ message RemoteJwks {
   //      uri: https://www.googleapis.com/oauth2/v1/certs
   //      cluster: jwt.www.googleapis.com|443
   //
-  api.v2.core.HttpUri http_uri = 1;
+  api.v2.core.HttpUri http_uri = 1 [(udpa.annotations.sensitive) = true];
```

Comment:
@yangminzhu @incfly both local and remote JWKS should be redacted, correct?

Reply:
Is this and also above really sensitive? I thought it's okay for the JWKS because it's only the public key and is supposed to be accessed by anyone who wants to verify the JWT token.

Reply:
If it's only the public key, then no, it's not sensitive (I wasn't sure if `JwtProvider` was used only for verification or also configuration, thanks for verifying).

I'll revert this change.